### ISSUE TITLE
No food at game start

### ISSUE THREAD

**Issue by sir-macelon** [aichallenge/aichallenge#229]
http://aichallengebeta.hypertriangle.com/visualizer.php?game=99254

**Comment by McLeopold** [comment]
According to the replay, food_start was 148, with the food_rate at 2 and food_turn at 13.  Can you try setting up a game with those values with that map to see if it happens again?

**Comment by sir-macelon** [comment]
How can I easily check what was the map name?
There's no such info in the replay.
There could be some link to the map on the game page to download it.
And maybe the "Game #XXX" link (on the game page) could be used for easier replay downloading instead of redirecting back to the same page?

**Comment by sir-macelon** [comment]
I've already found it is symmetric_38 map.
As I've checked it locally, after downloading latest source code from git, the problem is still there.
There is a food_visible parameter, which is specified in playgame.py, but it is not stored in replay.
If you set the food_visible to 3, then you get no food, otherwise you get 1 starting food per ant if you set food_visible to 2, and 2 starting food for food_visible = 1.
